libs/design-system/styles/tailwind.css

```diff
@@ -6,19 +6,10 @@
 @import './mdx';
 
 /* MDX styles. TODO: move to ./mdx.css and nest under .mdx-content */
```

Comment:
@neilZon could you comment these out instead? They'll have to be moved out to mdx.css and nested